### DISCUSSION TITLE
vscode editor no configuration options : Library Paths   available

### DISCUSSION THREAD

**Discussion by yaobo-lab** [slint-ui/slint#6864] [Q&A]
in linux vscode editor have a options  ： Library Paths
![企业微信截图_173215456633](https://github.com/user-attachments/assets/da773264-9141-4470-89a6-9b60bc4d520b)

Under the Win10 operating system, there is no
![image](https://github.com/user-attachments/assets/16fd50f3-3869-433c-8662-4cb4c5d17805)

Does this option not support the Windows system?

**Comment by tronical** [comment]
This option is also supported on Windows. Could it be that you have a different version of the extension installed? If you go to `Extensions`, enter `@installed slint`, then click on the Slint extension, what version is shown?

**Reply by yaobo-lab** [reply]
After upgrading vscode in Windows system ,it show  Library Paths  options .

Even after I configured it, the error still persists

![image](https://github.com/user-attachments/assets/a2de4186-a8db-435a-9045-e95f65bd78bd)

![image](https://github.com/user-attachments/assets/213fd427-8069-4a99-882a-cbc17d9270ca)

![image](https://github.com/user-attachments/assets/c9328c00-f1b8-4723-8b97-92732e59bfd7)

**Reply by hunger** [reply]
Can you try setting this in your `settings.json`:

```json
{
    "slint.libraryPaths": {
        "aimy": "g:/rust/code/libgui/aimy"
    }
}
```

I am not sure I am handling the backslash as a path separator correctly on Windows :-)

**Reply by ultimaweapon** [reply]
~~I also have the same issue but on Linux. Was tried both full path and relative path but none are working.~~ Strange, it works now.

**Reply by yaobo-lab** [reply]
thx
